script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -1472,7 +1472,7 @@ The Black Key lodged in the woman's right shoulder unleashes a jolt of electrici
 -- Page 508, Offset 28501.
 -- 殺傷より制圧に特化したソレは、<女|ノエル>の表面を焼かず、その<筋|う><肉|ち><組|が><織|わ>を徹底的に蹂躙した。
 The weapon, designed to subdue rather than eliminate,
- doesn't so much as scorch Noel's skin,
+ doesn't even scorch Noel's skin,
  but it thoroughly lays waste to her muscles underneath.
```

Comment:
Is the object of the rewrite the ordering? Honestly, looking at these 3, from a pure EN perspective the original reads  best to me still

Reply:
the relationship between the two clauses is unclear in EN, while it is very clear in JP, that's mostly it. "<女|ノエル>の表面を焼かず" is a qualifier of the upcoming sentence rather than it's own statement, sort of. That and the inside/outside doesn't come across well.

>thoroughly lays waste to the muscles inside of her without harming the skin outside.